.yarn/patches/@metamask-assets-controllers-patch-3424767802.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/dist/RatesController/RatesController.cjs b/dist/RatesController/RatesController.cjs
+index ecf10ece556eaa00f5ea967a8829574b22f29442..deb293761b54a60add7c4f7aa17130ae95187a79 100644
+--- a/dist/RatesController/RatesController.cjs
++++ b/dist/RatesController/RatesController.cjs
+@@ -84,6 +84,7 @@ class RatesController extends base_controller_1.BaseController {
+             return;
+         }
+         this.messagingSystem.publish(`${exports.name}:pollingStarted`);
++        await __classPrivateFieldGet(this, _RatesController_instances, "m", _RatesController_updateRates).call(this);
```

Comment:
On a fresh install; we have seen that its taking longer for the coin-overview component to update and show the user balance compared to the asset list.
That was because the conversion rate we get from this selector initally null 
```
  const conversionRate = useMultichainSelector(
    getMultichainConversionRate,
    account,
  );
```
In the current code when we call the start on the [multichainratesController](https://github.com/MetaMask/metamask-extension/blob/f73fd30e04702183fe00fa586bcfb55c879fe4a1/app/scripts/metamask-controller.js#L2936) : `this.multichainRatesController.start();`
Tt's taking around 3mins before we get the updated rates; so we added this line to trigger it.

TODO: We should refactor the multichainRatesController to extend `StaticIntervalPollingController`.

Reply:
I remember there being a reason for the controllers not being `StaticIntervalPollingController`. I think it was something to do with variable block times for different non-evm chains.

I agree that it would be best to all have the same pattern across our controllers.